migrations/sqls/20241116085102-initialize-up.sql

```diff
@@ -0,0 +1,275 @@
+-- back up with old schema
+alter schema badgehub rename to badgehub_old;
+create schema badgehub;
+create table badgehub.migrations
+(
+    like badgehub_old.migrations including all
+);
+
+-- create tables
+
+create table badges
+(
+    slug       text not null primary key,
+    name       text,
+    created_at timestamptz default now(), -- creation timestamp
+    updated_at timestamptz default now(), -- update timestamp
+    deleted_at timestamptz                -- soft delete timestamp (nullable)
+);
+
+create table users
+(
+    id                text primary key,          -- using text as recommended
+    email             text unique,               -- using text as recommended
+    admin             boolean,
+    name              text not null,
+    password          text not null,
```

Comment:
Are we salting these?

Reply:
Also no idea. But actually good point. Gonna remove the user api for now so that this code for user insertion does not get used in production  eventually accidentally.

Reply:
Ok I disabled the API in the next PR where I added it:
https://github.com/badgeteam/badgehub-api/pull/34/commits/dadfd316e6739d03b8f5f67163c1bd43a79d56ea